apps/sensenet/cypress/integration/breadcrumb/breadcrumb.spec.ts

```diff
@@ -0,0 +1,36 @@
+import { pathWithQueryParams } from '../../../src/services/query-string-builder'
+
+describe('Breadcrumb', () => {
+  before(() => {
+    cy.login()
+    cy.visit(pathWithQueryParams({ path: '/', newParams: { repoUrl: Cypress.env('repoUrl') } }))
+  })
+  it('breadcrumb after navigating to IT Workspace, breadcrumb should be displayed as it should.', () => {
+    const expectedBreadcrumbItems = ['Content', '/', 'IT Workspace']
+    cy.get('[data-test="drawer-menu-item-Content"]').click()
+    cy.get('[data-test="IT Workspace"]').click()
+    cy.get('nav[aria-label="breadcrumb"] li').each(($el) => {
+      expect(expectedBreadcrumbItems).to.include($el.text())
+    })
```

Comment:
Missing steps:

- Open IT workspace with clicking on it in the tree
- Expected result: children of IT Workspace are listed (Calendar is the first child item in the grid).
- Choose Document library from IT Workspace's children and open it
- Expected result: the breadcrumb contains the following items: Content, IT Worksapce, Document library

Reply:
should not "Expected result: children of IT Workspace are listed (Calendar is the first child item in the **grid**)." be "Expected result: children of IT Workspace are listed (Calendar is the first child item in the **list**)?? I can see Calendar in the list, not in the grid. please correct me if  i am wrong.

Reply:
@NileshPatel17,
You have to see them in both places (in the grid and in the list as well). But it is enough to check them in the grid regarding to the test case. See the attached gif.
![nileshp](https://user-images.githubusercontent.com/22347393/97963233-07157700-1db7-11eb-9b9e-cd3abac2d3ed.gif)